.eslintrc.json

```diff
@@ -1,16 +1,12 @@
 {
-    "env": {
-        "browser": true,
-        "es2020": true
-    },
-    "extends": [
-        "airbnb-base"
-    ],
-    "parserOptions": {
-        "ecmaVersion": 11,
-        "sourceType": "module"
-    },
-    "globals": {
+  "env": {
```

Comment:
Why did you update this file?

Reply:
I initially changed some stuff in the file when eslint stopped working before I realized it was a path issue.